### ISSUE TITLE
Meditation as a remedy for attention scarcity

### ISSUE THREAD

**Issue by erlend-sh** [WorldAfterCapital/WorldAfterCapital#36]
Meditation is touched upon briefly in the beginning of the book, but it’s never returned to. I think mediation as a topic here is worth exploring deeper. In a [recent interview](https://www.ted.com/talks/the_ted_interview_yuval_noah_harari_takes_us_into_the_past_and_reveals_the_real_dangers_ahead), Yuval Noah Harari points to meditation as one of the best tools at our disposal in the attention economy.

45:30 start
46:10 “you can’t understand what’s happening today in the world, with the attention economy, if you don’t observe what’s happening to your own attention, every moment”

1:04:50 “there is now a race, a competition, to hack humanity in general, and *you* in particular. And, you should make the effort, to stay ahead”
(continues to talk about meditation almost to the end of the interview)

**Comment by erlend-sh** [comment]
I just realised under "Psychological Freedom and Humanism" you do make a short case for meditation/mindfulness:

> Self-regulation, through some kind of mindfulness practice, lies at the heard of psychological freedom.

There's room for a bigger story here though.

**Comment by albertwenger** [comment]
Agreed. I will leave this comment open until I have done a better job of that. This will be part of my rewrite of Part Four (which is coming soon).
